js1-2-8.js

```diff
@@ -0,0 +1,19 @@
+let firstNumber = 7,
+    secondNumber = 10;
+
+console.log("The value of my variable foo is: " + firstNumber);
```

Comment:
foo = firstNumber

it's fine, just that you wrote `foo` instead of `firstNumber`:)